### ISSUE TITLE
Custom Command for feedback 

### ISSUE THREAD

**Issue by Gerrie12345** [bitfocus/companion-module-sony-visca#60]
There are some feedbacks in V2.7.0, I would like to use more like WB values and EXposure settings.
Is there a possibility to link a VISCA Inquiry to a Companion Variable?

**Comment by goseid** [comment]
Yep its on the roadmap and currently being worked on. You can expect this functionality in the beta builds in a couple of weeks and in the next release of Companion.

**Comment by Gerrie12345** [comment]
Are there still plans for the V2.8 update?

**Comment by goseid** [comment]
Closing this out — the module now polls the camera via VISCA inquiries and exposes 97+ variables including white balance mode, exposure mode, iris level, gain level, shutter speed, exposure compensation, and many more. There are also 16+ boolean feedbacks for WB mode, exposure mode, auto focus, backlight/spotlight compensation, tally, and others.

This was implemented in v2.8.0 and has continued to expand through v2.10.0. See the [HELP.md](https://github.com/bitfocus/companion-module-sony-visca/blob/master/companion/HELP.md) for the full list of variables and feedbacks available.
